testing/no_std/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "test-no-std"
+description = "Testing no_std build of scale-encode"
+readme = "README.md"
+
+version.workspace = true
+authors.workspace = true
+edition.workspace = true
+license.workspace = true
+repository.workspace = true
+homepage.workspace = true
+keywords.workspace = true
+include.workspace = true
+
+[dependencies]
+scale-encode = { path = "../../scale-encode", default-features = false, features = ["primitive-types", "derive", "bits"] }
```

Comment:
Because this is a part of the workspace I think it will still end up being executed with the "std" flag (because `scale-encode` itself will sue it, and features are combined across workspace crates I think).

It may be safer to add this instead to "workspace.exclude" to _not_ have it be a part of the workspace, to ensure no feature aggregation like this occurs (and then a separate test job to run it explicitly would be needed too) :)

Reply:
(I'd need to actually check this though to make sure I'm not wrong!)

Reply:
Ah, that was it.  Setting `workspace = true` indeed did not work - there was even a compiler warning that default-features is not available. However, I checked it multiple times to see if the no_std test crate is working - and it did notice no_std faults. But I can add the workspace exclude, should not be a problem.

Reply:
I just tested too and it seems to work ok to me; I geuss I need to read more on feature flag aggregating!